src/frontend/components/SiteHeader/index.tsx

```diff
@@ -34,17 +48,7 @@ export const SiteHeader: React.FC = () => {
       <TitleLink to="/" size="large" color="white">
         marsha.education
       </TitleLink>
-      {currentUser ? (
```

Comment:
IMO this is clearer than extracting a variable with a bunch of formal `if` clauses.